icu4c/source/data/misc/supplementalData.txt

```diff
@@ -35597,398 +35597,297 @@ supplementalData:table(nofallback){
-        }
-        TR{
-            consumption-vehicle-fuel{"liter-per-kilometer"}
-            length-person{"meter centimeter"}
-        }
-        UA{
-            concentr-blood-glucose{"millimole-per-liter"}
-        }
-        UG{
-            concentr-blood-glucose{"millimole-per-liter"}
+        "area"{
+            "geograph"{
+                001{
+                    square-kilometer{""}
+                }
+                GB{
+                    square-mile{""}
+                }
+                US{
+                    square-mile{""}
+                }
+            }
+            "land-agricult"{
+                001{
+                    hectare{""}
+                }
+                GB{
+                    acre{""}
+                }
+                US{
+                    acre{""}
+                }
+            }
+            "land-commercl"{
+                001{
+                    hectare{""}
+                }
+                GB{
+                    acre{""}
+                }
+                US{
+                    acre{""}
+                }
+            }
+            "land-residntl"{
+                001{
+                    hectare{""}
+                }
+                GB{
+                    acre{""}
+                }
+                US{
+                    acre{""}
+                }
+            }
+            _default{
+                001{
+                    square-kilometer{
+                        geq:1{""}
+                    }
+                    hectare{
+                        geq:1{""}
+                    }
+                    square-meter{
+                        geq:1{""}
+                    }
+                    square-centimeter{""}
+                }
+            }
         }
-        US{
-            area-land-agricult{"acre"}
-            area-land-commercl{"acre"}
-            area-land-residntl{"acre"}
-            consumption-vehicle-fuel{"mile-per-gallon"}
-            length-person{"inch"}
-            length-person-informal{"foot inch"}
-            length-person-small{"inch"}
-            length-rainfall{"inch"}
-            length-road{"mile"}
-            length-road-small{"foot"}
-            length-snowfall{"inch"}
-            length-vehicle{"foot inch"}
-            length-visiblty{"mile"}
-            length-visiblty-small{"foot"}
-            mass-person{"pound"}
-            mass-person-small{"pound ounce"}
-            pressure-baromtrc{"inch-hg"}
-            speed-road-travel{"mile-per-hour"}
-            speed-wind{"mile-per-hour"}
-            temperature-person{"fahrenheit"}
-            temperature-weather{"fahrenheit"}
-            volume-vehicle-fuel{"gallon"}
+        "duration"{
+            "music-track"{
+                001{
+                    minute+second{""}
+                }
+            }
+            "person-age"{
+                001{
+                    year-person+month-person{""}
```

Comment:
Shouldn't this one have a `geq` with bigger values being expressed only in years without months?